iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/extractor/dataregion/realtime/assigner/PipeDataRegionAssigner.java

```diff
@@ -73,6 +79,16 @@ public void assignToExtractor(
         .forEach(
             extractor -> {
               if (event.getEvent().isGeneratedByPipe() && !extractor.isForwardingPipeRequests()) {
+                // The frequency of progress reports is limited by the counter, while progress
+                // reports to TsInsertionEvent are not limited.
```

Comment:
TsFileInsertionEvent... And "The frequency of non-tsFile events' progress reports is limited by the counter."?

Reply:
TsFileInsertionEvent has only one implementation class, PipeTsFileInsertionEvent. Maybe it can be changed to TsFileInsertionEvent. For non-TsFileInsertionEvent, the reporting frequency needs to be limited.